plugins/rikki/loungestatistics/components/TeamStatistics.php

```diff
@@ -55,29 +55,27 @@ function ($group) {
             function ($season) {
                 return in_array($season->type, [1, 2]);
             }
-        )->
-            take(3)->pluck('title', 'id');
+        )->take(3)->pluck('title', 'id');
 
-        $this->selectedSeason = $this->participatedSeasons
-            ->filter(
-                function ($season) {
-                    return $season->is_active;
-                }
-            )->last() ?? $this->participatedSeasons->first();
+        $this->selectedSeasons = $this->participatedSeasons->filter(
```

Comment:
nit: This should get an array assigned (and can then be passed through in l 66)

Reply:
I split up that logic a bit, adding the array logic on top of the old one was a bit convoluted.

Reply:
I think the problem there was that the non-array ended up being persisted in $this->selectedSeasons at the end of the function. Looks good now though